.golangci.yml

```diff
@@ -36,6 +36,8 @@ linters-settings:
     ignore-words:
 
 issues:
+  exclude:
+    - Using the variable on range scope `tc` in function literal
```

Comment:
Required due to:
```
$ ./hack/verify-golint.sh
apis/v1alpha1/validation/validation_test.go:149:4: Using the variable on range scope `tc` in function literal (scopelint)
			tc.mutate(gw)
			^
apis/v1alpha1/validation/validation_test.go:151:11: Using the variable on range scope `tc` in function literal (scopelint)
			if len(tc.expectErrsOnFields) != len(errs) {
			       ^
apis/v1alpha1/validation/validation_test.go:152:59: Using the variable on range scope `tc` in function literal (scopelint)
				t.Fatalf("Expected %d errors, got %d errors: %v", len(tc.expectErrsOnFields), len(errs), errs)
```

xref: https://github.com/kyoh86/scopelint/issues/4#issuecomment-525540030

Reply:
can you get rid of the warning with a copy? 
`tc := tc`